server/pages/index.js

```diff
@@ -228,6 +230,7 @@ function setUpLoggedInRoute( req, res, next ) {
 			if ( error ) {
 				if ( error.error === 'authorization_required' ) {
 					debug( 'User public API authorization required. Redirecting to %s', redirectUrl );
+					res.clearCookie( 'wordpress_logged_in', { path: '/', httpOnly: true, domain: '.wordpress.com' } );
```

Comment:
We don't need to but I'd rather have it defined as it use to default to the current path for instance